shared/actions/chat.js

```diff
@@ -141,14 +138,24 @@ function * _incomingMessage (action: IncomingMessage): SagaGenerator<any, any> {
       const messageUnboxed: MessageUnboxed = incomingMessage.message
       const yourName = yield select(usernameSelector)
       const message = _unboxedToMessage(messageUnboxed, 0, yourName)
+      const conversationIDKey = conversationIDToKey(incomingMessage.convID)
+
+      // TODO short-term if we haven't seen this in the conversation list we'll refresh the inbox. Instead do an integration w/ gregor
+      const conversationStateSelector = (state: TypedState) => state.chat.get('conversationStates', Map()).get(conversationIDKey)
```

Comment:
Hmm. It's not documented, but I just tested in the console and this method seems to work. Not sure if it'll flowtype. The record type methods is one of my few frustrations with Immutable. :disappointed:  Not a big deal, we can disregard this suggestion.